### PR TITLE
Serhii Tolmachov. Added homework 1-3

### PR DESCRIPTION
**Pull request by SerhiyTo**

### CODE REVIEW

File: stolm/HW1/hw1_1.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+answer_name = input("What Is Your Name? - ")
```
Comment (kolyasalubov) [inline stolm/HW1/hw1_1.py:1]: good job

File: stolm/HW1/hw1_2.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+a = int(input("Enter A: "))
```
Comment (kolyasalubov) [inline stolm/HW1/hw1_2.py:1]: good job

File: stolm/HW3/hw3_1.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
+our_string = input("Enter your string: ")
+
```
Comment (kolyasalubov) [inline stolm/HW3/hw3_1.py:2]: good job

File: stolm/HW3/hw3_2.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
+number = int(input("Enter your number: "))
```
Comment (kolyasalubov) [inline stolm/HW3/hw3_2.py:1]: excellent job

File: stolm/HW3/hw3_2.py [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
```
Comment (kolyasalubov) [inline stolm/HW3/hw3_2.py:2]: please look at this implementation:
number = int(input("Enter your number: "))

one_div, remainder = divmod(number, 1000)
two_div, remainder = divmod(remainder, 100)
three_div, four_div = divmod(remainder, 10)
print(one_div, two_div, three_div, four_div)

#### Item 1: find of product of the digits of this number:

prod_of_dig = one_div * two_div * three_div * four_div
print("Product of the digits of this number:", prod_of_dig)

#### Item 2: write the number in reverse order:

str_number = str(number)[::-1]
print("Reverse of our number:", str_number)

#### Item 3: in ascending order, you need to sort the number included in the given number:

sort_number = ''.join(sorted(str(number)))

File: stolm/HW3/hw3_3.py [hunk 6]
```diff
@@ -0,0 +1,4 @@
+a = int(input("Enter number a: "))
```
Comment (kolyasalubov) [inline stolm/HW3/hw3_3.py:1]: good job

**Comment by SerhiyTo** [comment]
I added homework 4-6

File: stolm/HW4/hw4_1.py [hunk 7]
```diff
@@ -0,0 +1,7 @@
+cel = int(input("Enter your temperature in Celsius: "))
```
Comment (kolyasalubov) [inline stolm/HW4/hw4_1.py]: good job

File: stolm/HW5/hw5_1.py [hunk 8]
```diff
@@ -0,0 +1,11 @@
+N = int(input("Enter size of list: ")) # Get size of our list
```
Comment (kolyasalubov) [inline stolm/HW5/hw5_1.py:1]: good job

File: stolm/HW5/hw5_3.py [hunk 9]
```diff
@@ -0,0 +1,13 @@
+number = int(input("Enter your number: "))
```
Comment (kolyasalubov) [inline stolm/HW5/hw5_3.py]: good job

File: stolm/HW6/hw6_1.py [hunk 10]
```diff
@@ -0,0 +1,22 @@
+list = [ i for i in range(11) ]
```
Comment (kolyasalubov) [inline stolm/HW6/hw6_1.py:1]: good job

File: stolm/HW6/hw6_2.py [hunk 11]
```diff
@@ -0,0 +1,12 @@
+database = []
```
Comment (kolyasalubov) [inline stolm/HW6/hw6_2.py:1]: good job

File: stolm/HW5/hw5_2.py [hunk 12]
```diff
@@ -0,0 +1,17 @@
+number = int(input("Enter element for searching Fibonacci: "))
```
Comment (kolyasalubov) [inline stolm/HW5/hw5_2.py:1]: good job

File: stolm/HW5/hw5_3.py [hunk 13]
```diff
@@ -0,0 +1,13 @@
+number = int(input("Enter your number: ")) # Get the number 
+number_copy = number # Do copy of our number 
+
```
Comment (kolyasalubov) [inline stolm/HW5/hw5_3.py:3]: good job

File: stolm/HW7/hw7_1.py [hunk 14]
```diff
@@ -0,0 +1,18 @@
+def bigger(num1, num2):
```
Comment (kolyasalubov) [inline stolm/HW7/hw7_1.py:1]: good job

File: stolm/HW7/hw7_2.py [hunk 15]
```diff
@@ -0,0 +1,41 @@
+def area_rectangle(a, b):
```
Comment (kolyasalubov) [inline stolm/HW7/hw7_2.py:1]: good job

File: stolm/HW7/hw7_3.py [hunk 16]
```diff
@@ -0,0 +1,8 @@
+def counter_letters(word):
```
Comment (kolyasalubov) [inline stolm/HW7/hw7_3.py:1]: good job

File: stolm/HW7/hw7_codewars.py [hunk 17]
```diff
@@ -0,0 +1,85 @@
+# I. Jenny's secret message :
```
Comment (kolyasalubov) [inline stolm/HW7/hw7_codewars.py:1]: good job

File: stolm/HW8/HW8_1/main.py [hunk 18]
```diff
@@ -0,0 +1,4 @@
+from utils import *
```
Comment (kolyasalubov) [inline stolm/HW8/HW8_1/main.py:1]: good job

File: stolm/HW8/HW8_3/import_file.py [hunk 19]
```diff
@@ -0,0 +1,31 @@
+import math
```
Comment (kolyasalubov) [inline stolm/HW8/HW8_3/import_file.py:1]: good job

File: stolm/HW8/HW8_3/main.py [hunk 20]
```diff
@@ -0,0 +1,9 @@
+from import_file import display_menu
```
Comment (kolyasalubov) [inline stolm/HW8/HW8_3/main.py:1]: good job

File: stolm/HW8/HW8_2/HW8_2.py [hunk 21]
```diff
@@ -0,0 +1,26 @@
+def check_pass(password):
```
Comment (kolyasalubov) [inline stolm/HW8/HW8_2/HW8_2.py:1]: good job

look at this implementation

import re

def check_pass(password):
    # check length
    if not 6 <= len(password) <= 16:
        return "Your password should be between 6 and 16 characters long"

    # check for at least one uppercase letter, one lowercase letter, and one digit
    if not re.search(r"[A-Z]", password):
        return "Your password should contain at least one uppercase letter"
    if not re.search(r"[a-z]", password):
        return "Your password should contain at least one lowercase letter"
    if not re.search(r"\d", password):
        return "Your password should contain at least one digit"

    # check for at least one special character
    if not re.search(r"[$#@]", password):
        return "Your password should contain at least one of the following special characters: $ # @"

    # password is valid
    return "Your password is valid"

**Pull request merged**